### PR TITLE
 Checkbox/RadioButton components- suffix should be node (instead of string)

### PR DESCRIPTION
**Pull request by barboaron**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6771/

**Review by nitayneeman — COMMENTED** [review]
Looks good!

But two things that we missed out for both components:
* Testing - I think that visual tests are enough, for example we've [this](https://github.com/wix/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/Checkbox/Checkbox.visual.tsx#L59-L64) - so another one that passes node instead of string.
* Docs - Add an example with node suffix below [this](https://github.com/wix/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/Checkbox/docs/index.story.tsx#L63-L66) example.

We need to do the same for RadioButton but should be simple.

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6771/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6771

File: packages/wix-ui-tpa/src/components/Checkbox/Checkbox.visual.tsx [hunk 1]
```diff
@@ -58,9 +58,16 @@ visualize('Checkbox', () => {
...
     snap(
-      `${CheckboxTheme.Box} / suffix`,
+      `${CheckboxTheme.Box} / suffix (string)`,
       <CheckboxVisual theme={CheckboxTheme.Box} suffix="$50,000" />,
     );
+    snap(
+      `${CheckboxTheme.Box} / suffix (reactNode)`,
+      <CheckboxVisual
+        theme={CheckboxTheme.Box}
+        suffix={<span>$50,000</span>}
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Checkbox/Checkbox.visual.tsx:68]: Just to make it to something we can notice the difference :) 
```suggestion
        suffix={<span style={{backgroundColor: 'red', padding: '3px'}}>$10</span>}
```

File: packages/wix-ui-tpa/src/components/Checkbox/docs/examples.ts [hunk 2]
```diff
@@ -45,3 +45,9 @@ export const exampleWithSmallBoxAndSuffix = `
   <Checkbox theme="box" suffix="$10" onChange={val => console.log(val)} label="A lot of text, relatively." />
 </div>
 `;
+
+export const exampleWithSmallBoxAndReactNodeSuffix = `
+<div style={{ width: '250px' }}>
+  <Checkbox theme="box" suffix={<span>$10</span>} onChange={val => console.log(val)} label="A lot of text, relatively." />
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Checkbox/docs/examples.ts:51]:
```suggestion
  <Checkbox theme="box" suffix={<span style={{backgroundColor: 'red', padding: '3px'}}>$10</span>} onChange={val => console.log(val)} label="A lot of text, relatively." />
```

File: packages/wix-ui-tpa/src/components/Checkbox/docs/index.story.tsx [hunk 3]
```diff
@@ -64,6 +64,10 @@ const codeExamples = [
     title: 'With small box and suffix',
     source: examples.exampleWithSmallBoxAndSuffix,
   },
+  {
+    title: 'With small box and suffix',
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Checkbox/docs/index.story.tsx:68]:
```suggestion
    title: 'With small box and custom suffix',
```

**Review by nitayneeman — APPROVED** [review]
Approved, but added little enhancements :)

File: packages/wix-ui-tpa/src/components/Checkbox/docs/index.story.tsx (hunk 3 above)
Comment (barboaron) [reply packages/wix-ui-tpa/src/components/Checkbox/docs/index.story.tsx]: done

File: packages/wix-ui-tpa/src/components/Checkbox/docs/examples.ts (hunk 2 above)
Comment (barboaron) [reply packages/wix-ui-tpa/src/components/Checkbox/docs/examples.ts]: done

**Pull request merged**